builders/get-started/eth-compare/consensus-finality.md

```diff
@@ -42,63 +42,83 @@ However, the deterministic finality of Moonbeam can be used to provide a better
 
 The following sections outline how you can check for transaction finality using both the Ethereum JSON-RPC (custom Web3 request) and the Substrate (Polkadot) JSON-RPC.
 
-## Checking Tx Finality with Ethereum Libraries
-
-You can make calls to the Substrate JSON-RPC using the `send` method of both [Web3.js](https://web3js.readthedocs.io/) and [Ethers.js](https://docs.ethers.io/).
-
-Custom RPC requests are also possible using [Web3.py](https://web3py.readthedocs.io/) with the `make_request` method. You can use the Web3.js example as a baseline.
+## Checking Tx Finality with Moonbeam RPC Endpoints {: #checking-tx-finality-with-moonbeam-rpc-endpoints }
```

Comment:
I feel like this section shouldn't be duplicated... maybe just have the curl examples here and say `For more information you can go to the [Finality RPC Endpoints](/builders/build/moonbeam-custom-api#finality-rpc-endpoints){target=_blank} section of the Moonbeam Custom API page.` 

Not sure what the best path forward here is, if you have any ideas let me know! 🙂

Reply:
Good catch. I meant to revisit this part after splitting out the page.